tutorials/tutorial-01.qmd

```diff
@@ -6,33 +6,1403 @@ time: "~45 minutes"
+
+# Get European coastlines, transform to EPSG:3035, and crop to study area
+europe <- rnaturalearth::ne_countries(
+  scale = "medium",
+  continent = "Europe",
+  returnclass = "sf"
+) |>
+  sf::st_transform(3035) |>
+  sf::st_crop(bbox_3035)
+
+ggplot() +
+  geom_sf(data = europe, fill = "gray90", color = "gray60", linewidth = 0.3) +
+  geom_sf(data = bbox_3035, fill = NA, color = "#0077BE", linewidth = 1.5) +
+  labs(x = NULL, y = NULL) +
+  theme_minimal() +
+  theme(panel.grid = element_line(color = "gray85", linewidth = 0.2))
+```
+
+## Exploring EMODnet WFS Services
+
+Before downloading data, it's good practice to explore what's available. The `emodnet.wfs` package provides several functions to discover services, layers, and their attributes, helping you understand the data before committing to a download.
+
+### Discovering Available Services
+
+EMODnet provides data through several thematic portals. Let's see what [WFS services](../concepts.qmd#wfs-web-feature-service) are available:
+
+```{r}
+#| label: list-emodnet-services-code
+#| eval: false
+emodnet_wfs()
+```
+
+```{r}
+#| label: list-emodnet-services
+#| echo: false
+emodnet_wfs() |>
+  knitr::kable()
+```
+
+For this tutorial, we'll use the **human_activities** service, which contains both protected area boundaries and infrastructure data.
+
+### Connecting to a Service
+
+To query a service, we first create a WFS client connection:
+
+```{r}
+#| label: init-wfs-client
+wfs <- emodnet_init_wfs_client(service = "human_activities")
+```
+
+### Discovering Available Layers
+
+Each service contains multiple layers. Let's explore what's available:
+
+```{r}
+#| label: list-available-layers-code
+#| eval: false
+available_layers <- emodnet_get_wfs_info(wfs)
+available_layers |>
+  select(layer_name, title, abstract)
+```
+
+```{r}
+#| label: list-available-layers
+#| echo: false
+#| cache: true
+available_layers <- emodnet_get_wfs_info(wfs)
+available_layers |>
+  select(layer_name, title, abstract) |>
+  DT::datatable(
+    options = list(
+      pageLength = 7,
+      scrollX = TRUE,
+      columnDefs = list(
+        list(width = "150px", targets = 0),
+        list(width = "200px", targets = 1)
+      )
+    ),
+    rownames = FALSE
+  )
+```
+
+The full output also includes `service_name`, `service_url`, `class` (the WFS feature type), `format` (the R object class returned, typically `sf`), and `layer_namespace`.
+
+With many layers available, we can filter to find those relevant to our analysis.
+
+#### Protected Area Layers
+
+Let's search for layers related to protected areas:
+
+```{r}
+#| label: filter-protected-area-layers
+available_layers |>
+  filter(grepl("protected|natura|cdda", layer_name, ignore.case = TRUE)) |>
+  select(layer_name, title)
+```
+
+We'll use three complementary datasets to get comprehensive coverage of marine protected areas:
+
+- **`cddaareas`**: the CDDA database captures nationally designated sites, giving us country-level protected areas
```

Comment:
explain what the layer that we drop, the locations one, is?

Reply:
Good catch! Added an explanation. `cddalocations` contains point geometries while `cddaareas` has the polygon boundaries we need for spatial intersection.